packages/fxa-content-server/tests/functional/settings_v2/two_step_auth.js

```diff
@@ -0,0 +1,144 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+const intern = require('intern').default;
+const { describe, it, beforeEach } = intern.getPlugin('interface.bdd');
+const selectors = require('../lib/selectors');
+const FunctionalHelpers = require('../lib/helpers');
+const FunctionalSettingsHelpers = require('./lib/helpers');
+const {
+  navigateToSettingsV2,
+  confirmRecoveryCode,
+  generateSecurityCode,
+} = FunctionalSettingsHelpers;
+let securityCode;
+
+const {
+  click,
+  testElementExists,
+  type,
+  visibleByQSA,
+  testElementTextEquals,
+} = FunctionalHelpers.helpersRemoteWrapped;
+
+describe('two step auth', () => {
+  beforeEach(async ({ remote }) => {
+    await navigateToSettingsV2(remote);
```

Comment:
In addition to navigating, this also creates a new FxA user and sign in.  It's a bit wasteful in a `beforeEach` here, since you can use the same user to test enable and disable.  intern has a `before`, so you could create a user and nav once with that.

Reply:
Yes I was trying with before but when running the disable test somehow the same user and email was giving me error while logging in.  I struggled a lot with it and hence I wrote it in beforeEach using the helper functions so that I don't have to repeat the steps. I wrote the helper functions because I saw something similiar(not totally) with the existing helper functions confirmRecoveryCode, confirmTotpCode.

Reply:
> 
> 
> Yes I was trying with before but when running the disable test somehow the same user and email was giving me error while logging in. 

Why do you need to log in again?  The user's already logged in by the end of the enable test correct?  I think you just need to go back to the main settings page.

Reply:
okay! Sure. Let me work on this to incorporate your comments and I'll make the changes. Thanks for the thorough review! :)

Reply:
Having separate calls to create user and then navigate to new settings makes the test slightly more readable too.

Reply:
@vbudhram Is it okay for now to keep just that one function? if I split it into two now, I'll have to make changes to a few different tests as well and I was thinking to do it later after I am done with the 2FA tests.